docs/root/configuration/observability/access_log/usage.rst

```diff
@@ -679,6 +679,11 @@ UDP
   is unique with high likelihood within an execution, but can duplicate across
   multiple instances or between restarts.
 
+.. _config_access_log_format_upstream_connection_id:
+
+%UPSTREAM_CONNECTION_ID%
+  An identifier for the upstream connection.
+
```

Comment:
Thanks for you in timely update 😄 . This LGTM overall. Please supplement more context about the new command in the docs. I think this description should almost same with the `CONNECTION_ID` except it's used to tell the upstream connection id.

Reply:
done :)